inference-engine/src/gna_plugin/optimizer/gna_pass_manager.hpp

```diff
@@ -209,6 +209,11 @@ DECL_PASS(FuseFQIntoWeights);
 */
 DECL_PASS(MoveFakeQuantizeLayerIntoQuantParams);
 
+/**
+* @brief convert FullyConnected, ScaleShift and Eltwise layers weights order from NCHW to NHWC.
+* Information for transposition is found from convolution/pooling input or output dimensions.
+*/
+DECL_PASS(TransposeWeightsFromNCHWToNHWC);
```

Comment:
What about transposing weights (filters) of Convolution layer itself is it covered here or not?
And why/ why not ?

Reply:
Thanks, consider to add it as comment.